components/Administrator.vue

```diff
@@ -123,7 +136,9 @@ export default {
       this.validateForm();
     },
     validateForm() {
-      if(!this.$v.email.value.$invalid && !this.$v.password.value.$invalid && !this.$v.verify.value.$invalid) {
+      if(!this.$v.email.value.$invalid
+      && !this.$v.password.value.$invalid
```

Comment:
The && should be at the end of the previous line.  What you have here is not a linting error?

Reply:
Fixed linter issues.